addons/logistics_wirecutter/XEH_preInit.sqf

```diff
@@ -10,3 +10,29 @@ PREP(interactEH);
 PREP(isFence);
 
 ADDON = true;
+
+
+// Condition for Interaction
+DFUNC(Condition) = {
```

Comment:
They are 2 line functions that do not warrant their own global function. They belong with `interactEH`. Putting them in the `preInit` does not make sense to me; they would never be used anywhere else. Any potential performance gain you could get with it is mimimal at best.

I think that placing them in the `preInit` only makes the code structure harder to read in the long run.